initiatives/2020-05-initiatives-editor-raid.json

```diff
@@ -0,0 +1,49 @@
+[
+  {
+    "type": "sourcecred/initiativeFile",
+    "version": "0.2.0"
+  },
+  {
+    "title": "Initiatives Editor Raid",
+    "timestampIso": "2020-05-27T10:00:00.000Z",
+    "weight": {
+      "incomplete": 0,
+      "complete": 0
+    },
+    "completed": false,
+    "champions": [],
+    "dependencies": {},
+    "references": {
+      "urls": ["https://github.com/sourcecred/admin-dashboard"]
```

Comment:
Nit: Why is this called the admin dashboard? Rather than, say, "initiatives editor"?

Reply:
I was imagining that this could evolve into something more than just an initiatives editor. Even right now, we were thinking that it would be valuable to add new identities from a UI like this as well and setup the project configuration